emily/handler/src/api/models/deposit/requests.rs

```diff
@@ -91,3 +177,117 @@ pub struct UpdateDepositsRequestBody {
     /// Bitcoin transaction id.
     pub deposits: Vec<DepositUpdate>,
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use test_case::test_case;
+
+    const CREATE_DEPOSIT_VALID: &str =
+        include_str!("../../../../tests/fixtures/create-deposit-valid.json");
+
+    const CREATE_DEPOSIT_INVALID_TXID: &str =
+        include_str!("../../../../tests/fixtures/create-deposit-invalid-txid.json");
+
+    const CREATE_DEPOSIT_INVALID_OUTPUT_INDEX: &str =
+        include_str!("../../../../tests/fixtures/create-deposit-invalid-output-index.json");
+
+    const CREATE_DEPOSIT_INVALID_RECLAIM_SCRIPT: &str =
+        include_str!("../../../../tests/fixtures/create-deposit-invalid-reclaim-script.json");
+
+    const CREATE_DEPOSIT_INVALID_DEPOSIT_SCRIPT: &str =
+        include_str!("../../../../tests/fixtures/create-deposit-invalid-deposit-script.json");
+
+    const CREATE_DEPOSIT_INVALID_TRANSACTION_HEX: &str =
+        include_str!("../../../../tests/fixtures/create-deposit-invalid-transaction-hex.json");
+
+    const CREATE_DEPOSIT_MISMATCH_TXID: &str =
+        include_str!("../../../../tests/fixtures/create-deposit-mismatch-txid.json");
+
+    const CREATE_DEPOSIT_MISMATCH_RECLAIM_SCRIPT: &str =
+        include_str!("../../../../tests/fixtures/create-deposit-mismatch-reclaim-script.json");
+
+    const CREATE_DEPOSIT_MISMATCH_DEPOSIT_SCRIPT: &str =
+        include_str!("../../../../tests/fixtures/create-deposit-mismatch-deposit-script.json");
+
+    mod helpers {
+        use super::*;
```

Comment:
nit: I haven't seen those around, we usually just throw the helpers in the same `tests` module.